File: Perl/Files.pl
#!/usr/bin/perl
use strict;
=begin comment
  Some basic file operations here
=end comment
=cut

## 'cat' equivalent
my $file = '/etc/passwd';
open(PASSWD, $file);
my @lines = <PASSWD>;
close(PASSWD);
print "@lines";


## Creating a file
my $file_path = '~/Desktop/test.txt';
print "Creating a file $file_path";
sysopen(TEST, $file_path, "O_RDWR"|"O_EXCL"|"O_CREAT", 0755);
printf TEST "Testing\n";
close(TEST);

File: Python/Files.py
#!/usr/local/bin/python3

with open("test.txt", "a+") as file:
    file.write("Just a test file for python.\n")
    file.read()
    file.read(64)       # read 64 bytes
    file.readline()
    file.readlines()    # read all lines, can be exported to a list
